platform/platform-app/pom.xml

```diff
@@ -460,8 +467,8 @@
         <cxf.woodstox.core.version>${woodstox.core.version}</cxf.woodstox.core.version>
         <cxf.woodstox.stax2-api.version>${woodstox.stax2-api.version}
         </cxf.woodstox.stax2-api.version>
-        <cxf.xmlschema.version>2.2.1</cxf.xmlschema.version>
-        <cxf.wss4j.version>2.1.9</cxf.wss4j.version>
+        <cxf.xmlschema.version>2.2.2</cxf.xmlschema.version>
+        <cxf.wss4j.version>${wss4j.version}</cxf.wss4j.version>
```

Comment:
Sorry. Just saw something else. Since this version is now defined in the root pom, could you move the line
```xml
<wss4j.version>2.1.11</wss4j.version>
```
in the root pom to be in the section 
```xml
<!--START If these bundles are used elsewhere in DDF, the versions MUST exactly match the versions used by the CXF and Camel features.-->
        <woodstox.core.version>4.4.1</woodstox.core.version>
        <woodstox.stax2-api.version>3.1.4</woodstox.stax2-api.version>
        <xpp3.bundle.version>1.1.4c_6</xpp3.bundle.version>
        <jsr339-api.bundle.version>2.6.0</jsr339-api.bundle.version>
        <jaxb.version>2.2.11</jaxb.version>
        <dom4j.bundle.version>1.6.1_5</dom4j.bundle.version>
        <jdom.bundle.version>1.1_4</jdom.bundle.version>
        <geronimo-jms_2_spec.version>1.0-alpha-2</geronimo-jms_2_spec.version>
        <xalan.bundle.version>2.7.2_3</xalan.bundle.version>
        <!--END versions that MUST exactly match the versions used by the CXF and Camel features-->
```
so that the version defined in the DDF pom doesn't get out of sync with the cxf version?

Reply:
Sure, didn't realize we had a section for those properties.

Reply:
Fixed.